.gitignore

```diff
@@ -1,4 +1,4 @@
-
+env/
```

Comment:
Probably should add `package-lock.yml`.

Reply:
The [package-lock.yml](https://github.com/fivetran/dbt_qualtrics/blob/97d0b08b8d3b7ffbeb10e7a6ba46ce73385da93d/.gitignore#L5) is already included in the .gitignore.